.github/workflows/depcheck.yml

```diff
@@ -0,0 +1,27 @@
+name: Depcheck
+
+on:
+  push:
+    branches:
+      - nextjs
+  pull_request:
+    branches:
+      - nextjs
+
+jobs:
+  check_dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+
+      - name: Install depcheck
+        run: npm install -g depcheck
```

Comment:
sweep: how do you know that this is correct? Can you point me to some documentation to confirm

Reply:
sweep: I don't see a hyperlink. Cmon man, can you please search the web and link me a URL

Reply:
sweep: That link doesn't work. Sheesh, I have to do the work for you I guess

https://www.npmjs.com/package/depcheck

Reply:
sweep: we also use yarn as our package manager. Can you update the github action file (`.github/workflows/depcheck.yml`) accordingly?